.travis.yml

```diff
@@ -8,7 +8,9 @@ cache:
   - pip
 
 install:
-  - pip install .
+  - pip install -r requirements.txt
+  - pip install -r test_requirements.txt
+  - pip install pandas
```

Comment:
why have this here, and not in requirements.txt?

Reply:
@kiddinn this PR is WIP, I've not added a reviewer yet. The reason why this is here since pandas is here because of the previous question.